CMakeLists.txt

```diff
@@ -102,6 +121,26 @@ if( HAVE_NVTX )
     list( APPEND ECRAD_COMPILE_DEFINITIONS HAVE_NVTX )
 endif()
 
+if (HAVE_GPU)
+  set(HAVE_HIP 0)
+  if(CMAKE_VERSION VERSION_GREATER_EQUAL "3.24")
```

Comment:
I think we'd be fine to alternatively increase the CMake version requirement to 3.24 on l. 10 - we're demanding 3.20 already (~4y old) and 3.24 is not that much more recent (~3y old)

Reply:
Fixed